crypto/abi_self_test.cc

```
@@ -344,79 +344,79 @@ TEST(ABITest, ARM) {
+void abi_test_clobber_x1();
+void abi_test_clobber_x2();
+void abi_test_clobber_x3();
+void abi_test_clobber_x4();
+void abi_test_clobber_x5();
+void abi_test_clobber_x6();
+void abi_test_clobber_x7();
+void abi_test_clobber_x8();
+void abi_test_clobber_x9();
+void abi_test_clobber_x10();
+void abi_test_clobber_x11();
+void abi_test_clobber_x12();
```

Cppcheck (2.19.0)

[style] The function 'Check' is never used.

(unusedFunction)